docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -77,6 +77,21 @@ function App() {
 
 To use the `v5` version of MUI Core, you first need to update the package names:
 
+```sh
+npm install @mui/material @mui/styles
+
+// or with `yarn`
+yarn add @mui/material @mui/styles
+```
+
+**Optional**: if you have one these packages, install the new package separately
+
+- For `@material-ui/lab`, install `@mui/lab`
+- For `@material-ui/icons`, install `@mui/icons-material`
+
+<details>
+<summary>See all packages change</summary>
```

Comment:
With this collapse, a developer wouldn't find what happened with `@material-ui/core` by searching on the page. I would advocate that `@material-ui/core -> @mui/material` should be made obvious, not implicit.

Reply:
> With this collapse, a developer wouldn't find what happened with @material-ui/core by searching on the page

Fair enough, I can remove the collapse.

> "you first need to update the package" Ok, but what goes where?

I think this should change to "To use the `v5` version of MUI Core, install these packages"

Reply:
and mention that the package names have changed (I'm sure that some people will be surprised)